primitives/log/log.go

```diff
@@ -0,0 +1,22 @@
+//go:build !nonwasmenv
+
+package log
+
+import (
+	"github.com/LimeChain/gosemble/env"
+	"github.com/LimeChain/gosemble/utils"
+)
+
+const (
+	Critical = iota
+	Warn
+	Info
+	Debug
+	Trace
+)
+
+func Log(level int32, target []byte, message []byte) {
+	targetOffsetSize := utils.BytesToOffsetAndSize(target)
+	messageOffsetSize := utils.BytesToOffsetAndSize(message)
+	env.ExtLoggingLogVersion1(level, targetOffsetSize, messageOffsetSize)
+}
```

Comment:
This should be private. Usually loggers have different funcs depending on the level of logging.
Create public `func Critical, Warn, Info, Debug, Trace`.

Example:
```
func Warn(target, message []byte) {
  log(Warn, target, message)
}
```

Usually, when loggers log with level `Fatal`, which we can consider to `Critical`, they exit the program.